deploying-grr-clients/on-windows.md

```diff
@@ -66,7 +66,7 @@ execute it. The installation doesn’t require user input.
 
 The expected output is something along these lines:
 
-```docker
+``docker
```

Comment:
qq: is this intended? Typically triple accent is used for code blocks. The same block is also closed with triple.

Reply:
That's accidental - thanks for spotting!